README.md

```diff
@@ -283,8 +283,9 @@ And finally, you are ready to go! Or at least you should be...
 
 ### Junior Contributors
 
+**Eira Fontanals** | [eirafontanals][eirafontanals] | I am an easy laughter person.
 
-
+[eirafontanals]: <https://github.com/eirafontanals>
 
```

Comment:
If you could, place this link in its proper line following the alphabetical order.